expo/expo
# [expo-modules-autolinking] Extend @tsconfig/node12 base config

# Why

This fixes one of the issues from #13475 - we were building expo-modules-autolinking with the `esnext` target, which requires Node 14+. Our installation instructions any of the LTE releases, and 12 is one of those, so we should support it.

# How

Switch to the @tsconfig/node12 base config provided by the TypeScript team

# Test Plan

- Init a bare project
- Switch to Node 12
- Run `yarn android` in the project, see it fail
- Copy build dir from this pr into project node_modules
- Run `yarn android` in the project, rejoice in great success

# Checklist
